source/includes/administration/_custom_navigation.md

```diff
@@ -0,0 +1,282 @@
+### List custom navigation
+
+`GET /custom_navigation`
+
+```shell
+
+curl --request GET \
+  --url http://cloudmc_endpoint/rest/navigation/:id \
+  --header 'Content-Type: application/json' \
+  --header 'Mc-Api-Key: your_api_key'
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "reseller": {
+        "id": "c869e848-6fb3-4850-af3d-42c5666f2c78"
+      },
+      "id": "f8663571-1d84-4d48-bb86-d4e2a14ea9f9",
+      "enabled": true
+    }
+  ]
+}
+```
+
+Retrieves a list of custom navigations. Since there is only one custom navigation per reseller this list will either be of length 1 or 0.
+
+Query Params | &nbsp;
+---- | -----------
+`organization_id`<br/>*UUID* | Return only the custom navigation for the provided organization id. If not provided, will default to the calling user's organization. Must be a reseller organization.
+
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The custom navigations's id.
+`reseller.id`<br/>*UUID* | The organization id that the custom field belongs to.
+`enabled`<br/>*boolean* | Whether not custom navigation is enabled on the reseller.
+
+<!-------------------- Get CUSTOM NAV -------------------->
+### Get custom navigation
+
+`GET /custom_navigation/:id`
+
+
+```shell
+
+curl --request GET \
+  --url http://cloudmc_endpoint/rest/navigation/f8663571-1d84-4d48-bb86-d4e2a14ea9f9 \
+  --header 'Content-Type: application/json' \
+  --header 'Mc-Api-Key: your_api_key'
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "reseller": {
+      "id": "c869e848-6fb3-4850-af3d-42c5666f2c78"
+    },
+    "tabs": [
+      {
+        "name": {
+          "en": "system en",
+          "fr": "system fr"
+        },
+        "rank": 0,
+        "id": "e4f527bf-9de6-4055-9ede-6949d01156f9",
+        "tag": {
+          "en": "tag en",
+          "fr": "tag fr"
+        },
+        "type": "SYSTEM",
+        "key": "/users"
+      },
+      {
+        "name": {
+          "en": "system en",
+          "fr": "sytem fr"
+        },
+        "rank": 1,
+        "id": "125e296a-ae59-4cd7-8d5b-0cf354cbb1cc",
+        "type": "COMING_SOON"
+      }
+    ],
+    "id": "f8663571-1d84-4d48-bb86-d4e2a14ea9f9",
+    "enabled": true
+  }
+}
+```
+
+Retrieves a detailed custom navigation by id.
+
+Query Params | &nbsp;
+---- | -----------
+`organization_id`<br/>*UUID* | Return only the custom navigation for the provided organization id. If not provided, will default to the calling user's organization. Must be a reseller organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The custom navigations's id.
+`reseller.id`<br/>*UUID* | The organization id that the custom field belongs to.
+`enabled`<br/>*boolean* | Whether not custom navigation is enabled on the reseller.
+`tabs`<br/>*Array[Object]* | List of tabs, the the order they should be displayed in a the menu.
```

Comment:
Can remove the bit about order since I've seen things come back out of order (maybe bcs I made changes directly in database)